addon/helpers/d3-array.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import array from 'd3-array';
+
+let allowedMethods = ['min', 'max', 'extent', 'sum', 'mean', 'median', 'variance', 'deviation', 'scan', 'merge', 'pairs', 'permute'];
+export function d3Array([ method, ...args ]) {
+  Ember.assert('this method is not yet supported', Ember.A(allowedMethods).contains(method));
```

Comment:
oh, good catch @spencer516 - we were using `includes`, but phantomJS was unhappy with that, hence my addition of the Ember array - I will move the array instantiation outside of the helper in https://github.com/LocusEnergy/ember-d3-helpers/pull/8